### PR TITLE
Changelog for edge-19.11.1

### PR DESCRIPTION
**Pull request by ihcsim**
Signed-off-by: Ivan Sim <ivan@buoyant.io>

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,28 @@
+## edge-19.11.2
+
+* CLI
+  * Updated `uninject` command to uninject namespace resources
```
Comment (zaharidichev) [inline CHANGES.md]: TIOLI: maybe avoid the second "uninject" and just do: `to work with namespace resources`. We already can tell  that the purpose of the command is to uninject things :)

File: CHANGES.md [hunk 2]
```diff
@@ -1,3 +1,28 @@
...
+    (thanks @mayankshah1607!)
+* Controller
+  * Added `conntrack` to the `debug` container to help with connection tracking
```
Comment (zaharidichev) [inline CHANGES.md]: TIOLI: Somehow `connection tracking debugging` feels like a mouthful. What about `to allow/enable connection tracking`

File: CHANGES.md [hunk 3]
```diff
@@ -1,3 +1,28 @@
...
+    debugging
+  * Fixed a bug in `tap` where mismatch cluster domain and trust domain caused
```
Comment (zaharidichev) [inline CHANGES.md]: TIOLI: `caused tap to hang` -> `caused it/the cli to hang`

File: CHANGES.md [hunk 4]
```diff
@@ -1,3 +1,28 @@
...
+    `tap` to hang
+  * Fixed an issue in the `identity` RBAC resource which caused start up errors
+    in k8s 1.6 (thanks @Pothulapati!)
+* Proxy
+  * Improved debug/error logging to include a wealth of contextual information
```
Comment (olix0r) [inline CHANGES.md]: i know these are my words but let's `s/a wealth of/detailed/`

File: charts/linkerd2/values.yaml [hunk 5]
```diff
@@ -7,7 +7,7 @@ EnableH2Upgrade: true
 ImagePullPolicy: &image_pull_policy IfNotPresent
 
 # control plane version. See Proxy section for proxy version
-LinkerdVersion: &linkerd_version edge-19.10.5
+LinkerdVersion: &linkerd_version edge-19.11.2
```
Comment (olix0r) [inline charts/linkerd2/values.yaml]: Hm? This should be 11.1?

File: CHANGES.md [hunk 6]
```diff
@@ -1,3 +1,28 @@
+## edge-19.11.2
```
Comment (olix0r) [inline CHANGES.md]: Isn't this 11.1?

**Pull request merged**